client/components/Mentions.vue

```diff
@@ -13,7 +13,7 @@
 					class="btn hide-all-mentions"
 					@click="hideAllMentions()"
```

Comment:
Not sure if we should change the terminology from "hide" to "dismiss" across the codebase too

Reply:
https://github.com/thelounge/thelounge/pull/4322/commits/90df2951ac94299e0cb36962011314e1dd7f6127